understanding/22/accessible-authentication-minimum.html

```diff
@@ -67,7 +66,7 @@ <h2>In brief</h2>
     </section>
 
     <section id="intent">
-        <h2>Intent of Accessible Authentication</h2>
+        <h2>Intent of Accessible Authentication (Minimum)</h2>
```

Comment:
As per prior comment, not sure if they're being stripped by scripting or what, I but I do not see SC titles as part of sub-headings. It's normally just "Intent", "Benefits", etc